ccd-definition/AuthorisationCaseEvent/CareSupervision/AuthorisationCaseEvent-nonprod.json

```diff
@@ -124,5 +124,12 @@
     "CaseEventID": "populateCase-PREPARE_FOR_HEARING",
     "UserRole": "caseworker-publiclaw-courtadmin",
     "CRUD": "R"
+  },
+  {
+    "LiveFrom": "01/01/2017",
+    "CaseTypeID": "CARE_SUPERVISION_EPO",
+    "CaseEventID": "editFamilyManCaseNumber",
```

Comment:
does any other user need to see this event? (in history tab)? I'm guessing not, just wanted to double check. Probably fine